content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -141,6 +141,10 @@ ECS tasks that run on Fargate are already fully allocated [in the CUR][103]. CCM
 
 For Kubernetes data transfer allocation, a Kubernetes node is joined with its associated data transfer costs from the [CUR][103]. The node's cluster name and all node tags are added to the entire data transfer cost for the node. This allows you to associate cluster-level dimensions with the cost of the data transfer, without considering the pods scheduled to the node.
 
+Next, Datadog looks at all of the [workload resources][104] running on that node for the day. The cost of the node is allocated to the workload level based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the workload resource's tags.
```

Comment:
do we want to be more specific about "resources" -- e.g. based on the amount of bits etc

Reply:
yes, added more detail that we use volume of network traffic to allocate costs